src/render/OpenGL.cpp

```diff
@@ -511,6 +520,14 @@ void CHyprOpenGLImpl::renderTextureInternalWithDamage(const CTexture& tex, wlr_b
     glUniformMatrix3fv(shader->proj, 1, GL_FALSE, glMatrix);
 #endif
     glUniform1i(shader->tex, 0);
+
+    if (usingFinalShader && g_pConfigManager->getInt("debug:damage_tracking") == 0) {
+        glUniform1f(shader->time, m_Timer.getSeconds());
```

Comment:
this is not an issue per se - but wouldn't milliseconds with an int make more sense here? Dunno if opengl uses 64 or 32 bit ints though - if 32 then it's not really a good solution imho (overflow after 596h) but if it's 64-bit we can make it have zero inaccuracies.

Reply:
Good question. In terms of shader programming, ```float time;``` in seconds would be way less awkward than ```int time;``` in milliseconds and definitely more common. We're limited anyways by the return value of the internal call to ```std::...::duration::count()``` (which is a 32bit long and in milliseconds for ```getSeconds```). ```GLint``` is also typedef'ed to ```int``` on the C++ side.

Reply:
> We're limited anyways by the return value of the internal call to `std::...::duration::count()` (which is a 32bit long and in milliseconds for getSeconds).

umm...
![image](https://user-images.githubusercontent.com/43317083/222965169-515c4769-0a34-44b6-945d-cf70f40e9d25.png)

> `GLint` is also typedef'ed to `int` on the C++ side (GLlong doesn't exist and IIRC 64bit integers don't exist for OpenGL).

Right. Forgot.

Well then this is alr.